src/ng/components/body/body.core.js

```diff
@@ -58,41 +72,62 @@ class BodyCore extends Directive(BODY_CORE_NAME, {view: `^^${VIEW_CORE_NAME}`})
 		}
 	}
 
-	toggle(cell) {
+	onMouseDown(e) {
+		if (!this.isRange) {
+			return;
+		}
+
+		this.rangeStart = e;
```

Comment:
incapsulate e please, it looks like overlay is a usefull common component, let's apply more explicit contract for it methods

this.rangeStart = {x: e.pageX, y: y.pageY}